src/open_inwoner/userfeed/adapters.py

```diff
@@ -0,0 +1,40 @@
+from open_inwoner.userfeed.adapter import FeedItem
+
+
+def get_item_adapter_class(type: str) -> type[FeedItem]:
+    return feed_adapter_map.get(type, feed_adapter_default)
+
+
+feed_adapter_default = FeedItem
+feed_adapter_map = dict()
+
+
+def register_item_adapter(adapter_class: type[FeedItem], feed_type: str):
```

Comment:
Probably a bit nitpicky, but would we want to enforce that `feed_type` is one of the options in the `FeedItemType` choices?

Reply:
Yes we would, but I'll have to look into how we'd do that (and if python 3.9(

Reply:
Ugh, apparently we can just use the type of the choices class instead of `str`.